x/erc20/types/utils_test.go

```diff
@@ -13,17 +15,26 @@ func TestSanitizeERC20Name(t *testing.T) {
 		erc20Name    string
 		expErc20Name string
 	}{
-		{"name contains ' Token'", "Lucky Token", "lucky"},
-		{"name contains ' Coin'", "Otter Coin", "otter"},
-		{"name contains ' Token' and ' Coin'", "Lucky Token Coin", "lucky"},
-		{"multiple words", "Hextris Early Access Demo", "hextris_early_access_demo"},
-		{"single word name: Token", "Token", "token"},
-		{"single word name: Coin", "Coin", "coin"},
+		{"name contains 'Special Characters'", "*Special _ []{}||*¼^%  &Token", "SpecialToken"},
+		{"name contains 'Spaces'", "   Spaces   Token", "SpacesToken"},
+		{"name contains 'Leading Numbers'", "12313213  Number     Coin", "NumberCoin"},
+		{"name contains 'Numbers in the middle'", "  Other    Erc20 Coin ", "OtherErc20Coin"},
+		{"name contains '/'", "USD/Coin", "USD/Coin"},
```

Comment:
I think it's important that we prevent ERC20s to be registered that start with `ibc/` or `erc20/`

Reply:
those are registered namespaces in the cosmos denoms